Data 23

andrey-canon opened the pull request:
## Description 
Get for vertical component section and subsection data, generate general completion report per unit or vertical component.

Squirrel18 commented on report_backend/completion_report.py:
```diff
@@ -36,27 +37,46 @@ def json_report_to_csv(self, json_report_data={}):
+                    unit = general_course_data.get(component.get('number'))
+
+                    if not unit:
+                        od = OrderedDict()
```
Could you please chnage the variable name. @andrey-canon

andrey-canon committed:
Addressing PR comments
Get for vertical component section and subsection data, generate general completion report per unit or vertical component,
